regression/cbmc-java/reachability-slice/test2.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+A.class
+--reachability-slice-fb --show-goto-functions --property "java::A.foo:(I)V.coverage.3" --cover location
+1001
+1002
+1003
+1005
```

Comment:
Looking at the tests failing on AppVeyor, I am wondering whether `--show-goto-functions` actually does not produce any output on Windows.